### PR TITLE
feat(icons): add new icons to the font

### PR DESCRIPTION
**Pull request by silviyaboteva**

### CODE REVIEW

File: packages/default/scss/icons/_layout.scss [hunk 1]
```diff
@@ -283,6 +283,7 @@
     .k-i-gap-row::before { content: "\e05b"; }
     .k-i-handle-resize-alt::before { content: "\e05c"; }
     .k-i-handle-resize::before { content: "\e05d"; }
+    .k-i-shevron-top::before { content: "\e94a"; }
```
Comment (joneff) [inline packages/default/scss/icons/_layout.scss:286]:
```suggestion
    .k-i-chevron-double-up::before { content: "\e94a"; }
```

File: packages/default/scss/icons/_layout.scss [hunk 2]
```diff
@@ -1054,6 +1055,8 @@
     .k-i-files-validation::before { content: "\e945"; }
     .k-i-data-rest::before { content: "\e946"; }
     .k-i-file-typescript::before { content: "\e947"; }
+    .k-i-data-ods::before { content: "\e948"; }
+    .k-i-report::before { content: "\e949"; }
```
Comment (joneff) [inline packages/default/scss/icons/_layout.scss:1059]:
```suggestion
    .k-i-file-report::before { content: "\e949"; }
```

**Review by joneff — CHANGES_REQUESTED** [review]
we need the three additional double chevrons

**Comment by joneff** [comment]
as an additional note, we need to move the icons to kendo-icons repo and use it to generate those. This particular PR is going to take a bit more

File: packages/default/scss/icons/_layout.scss (hunk 1 above)
Comment (silviyaboteva) [reply packages/default/scss/icons/_layout.scss:286]: Synced with Pavel. He will add the rest three icons and will rename them to `k-i-chevron-double-up|down|left|right`. Once he push them, I'll continue :)

**Comment by silviyaboteva** [comment]
I'll open a separate PR in `kendo-icons` repo.

**Review by joneff — APPROVED** [review]

**Pull request merged**